boards/arm/mec1501modular_assy6885/doc/index.rst

```diff
@@ -69,7 +69,7 @@ features:
 Other hardware features are not currently supported by Zephyr (at the moment)
 
 The default configuration can be found in the
-:zephyr_file:`boards/arm/mec1501modular_assy6885/mec1501modular_assy6885`
+:zephyr_file:`boards/arm/mec1501modular_assy6885/Kconfig.defconfig`
```

Comment:
This should be `mec1501modular_assy6885_defconfig`

Reply:
Thanks.  I took a guess :)